utils/check_copies.py

```diff
@@ -494,6 +494,7 @@ def check_model_list_copy(overwrite=False, max_per_line=119):
     "Data2VecVision": "Data2Vec",
     "DonutSwin": "Donut",
     "Marian": "MarianMT",
+    "MaskFormerSwin": "MaskFormer",
```

Comment:
Might be more logical to put `Swin` here since it's a variant of it?
Just a nit.

Reply:
When I do that, it adds MaskFormerSwin to the 8 READMEs.

Reply:
The exact name would be "Swin Transformer"

Reply:
Oh thanks, that worked!